File: templates/griffon-javafx-java-templates/templates/subtmpl-artifact/FunctionalTest.java
package ${project_package};

import griffon.test.javafx.FunctionalJavaFXRunner;
import griffon.test.javafx.GriffonTestFXClassRule;
import org.junit.ClassRule;
import org.junit.Test;
import org.junit.runner.RunWith;

@RunWith(FunctionalJavaFXRunner.class)
public class ${project_class_name}FunctionalTest {
    @ClassRule
    public static GriffonTestFXClassRule testfx = new GriffonTestFXClassRule("mainWindow");

    @Test
    public void smokeTest() {
        fail("Not yet implemented!");
    }
}

File: templates/griffon-javafx-kotlin-templates/templates/subtmpl-artifact/FunctionalTest.kt
package ${project_package}

import griffon.test.javafx.FunctionalJavaFXRunner
import griffon.test.javafx.GriffonTestFXClassRule
import org.junit.Assert
import org.junit.ClassRule
import org.junit.Test
import org.junit.runner.RunWith

@RunWith(FunctionalJavaFXRunner::class)
class ${project_class_name}FunctionalTest {
    companion object {
        @ClassRule @JvmField
        val testfx: GriffonTestFXClassRule = GriffonTestFXClassRule("mainWindow")
    }

    @Test
    fun smokeTest() {
        Assert.fail("Not yet implemented!")
    }
}
